intro.ipynb

```diff
@@ -1,99 +1,179 @@
+    "\n",
+    "### On Zoom\n",
+    "- Raise your hand 🤚 then ask\n",
+    "- Write in the chat\n",
+    "- We will open a break-out room to address longer questions. **Anyone** interested can jump in\n",
+    "\n",
+    "\n",
+    "### On Slack\n",
+    "- Ask your question in the **corresponding channel**. Use `#help` for generic questions\n",
+    "- Reply **in a thread** if you want to add a comment on another question\n",
+    "- Use `@Sasha`, `@Simone`, `@Edoardo` if you want to mention the tutorial instructors directly\n",
+    "\n",
+    ":::\n",
+    "\n",
+    "::: {.notes}\n",
+    "\n",
+    "Although the chat will be distributed together with the recording, Slack is much better suited to hold continued Q&A and other discussions. Please, use it.\n",
+    "\n",
+    ":::"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "q1 = Question(\n",
-    "    question='What is the answer to life, the universe, and everything?',\n",
-    "    options={\n",
-    "        '42': 'The answer is 42! You are correct!',\n",
-    "        '41': 'A bit more',\n",
-    "        '44': 'A bit less',\n",
-    "        '45': 'A bit less',\n",
-    "        },\n",
-    "    hint=\"2**2 * 10 + 4\",\n",
-    "    correct_answer='42',\n",
-    "    shuffle=True,\n",
-    "    )\n",
-    "\n",
-    "q2 = Question(\n",
-    "    question=\"How many planets are there in our solar system?\",\n",
-    "    options={\n",
-    "        '8': 'The answer is 8. You are correct!',\n",
-    "        '9': 'You are close, but not quite there.',\n",
-    "        '10': 'A bit less.',\n",
-    "    },\n",
-    "    hint=\"pluto is not a planet anymore.\",\n",
-    "    correct_answer='8',\n",
-    "    shuffle=True,\n",
-    "    )\n",
-    "\n",
-    "quiz = Quiz(questions=[q1, q2])\n",
+    "## How to use notebooks\n",
+    "\n",
     "\n",
+    "1. The link to open Binder is on Slack in `#announcements` and here in the chat\n",
+    "2. Open the `index.ipynb` file and click on the current topic link\n",
+    "3. Run the code, modify, experiment\n",
     "\n"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [],
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exercises\n",
+    "\n",
+    "::: {.fragment}\n",
+    "\n",
+    "For each topic we prepared a bunch of exercises to practice the concepts\n",
+    "\n",
+    ":::\n",
+    "\n",
+    "::: {.fragment}\n",
+    "\n",
+    "You will see cells like\n",
+    "\n",
+    "```python\n",
+    "%%ipytest\n",
+    "\n",
+    "def solution_to_exercise(input_arg):\n",
+    "    \"\"\"\n",
+    "    Write your solution here\n",
+    "    \"\"\"\n",
+    "    pass\n",
+    "```\n",
+    "\n",
+    "**Unless otherwise noted**, you can delete everything after the line starting with `def` and write your solution.\n",
```

Comment:
Let's add a reminder that they need to run the cell containing `%load_ext tutorial.tests.testsuite` before the exercises.

Reply:
Ok, I'll do that, as discussed